book/06-github/sections/2-contributing.asc

```diff
@@ -123,7 +123,7 @@ To https://github.com/tonychacon/blink
 
 Now if we go back to our fork on GitHub, we can see that GitHub noticed that we pushed a new topic branch up and presents us with a big green button to check out our changes and open a Pull Request to the original project.
 
-You can alternatively go to the "`Branches`" page at `https://github.com/<user>/<project>/branches` to locate your branch and open a new Pull Request from there.
+You can alternatively go to the "`Branches`" page at `\https://github.com/<user>/<project>/branches` to locate your branch and open a new Pull Request from there.
```

Comment:
Is ``` `h ``` some kind of special code in Asciidoc? What was the error code here?

Reply:
With the new syntax of asciidoctor, the backticks no longer convey a semantic format, but are just mere font changes. So they no longer escape other formating inside the ticks. Here the "https://" form is detected and converted into a link, which brings three issues:

* is a url without the trailing `[]` supposed to be converted
* This is not a real URL so converting is undesirable
* Even if we let it converted, the conversion has a bug which does not handle the < and > entities correctly and produces invalid xhtml code.

Putting a backslash just disables URL scheme detection and lets the backticks do the font change.

Reply:
Replying to first point: this is a feature called "autolink".

Reply:
For Ref: Please see https://github.com/asciidoctor/asciidoctor/issues/3864

Reply:
Awesome, thanks for the info.